### PR TITLE
Added more info on how to do a local release

### PR DESCRIPTION
**Pull request by obigu**
Some people on Discord have asked recently and in the past how to make a local release. I've tried to organize it in sections to improve clarity, suggestions/corrections are welcome.

### CODE REVIEW

File: _pages/dev/-contributing/from_source.md [hunk 1]
```diff
@@ -86,13 +86,43 @@ GL30 emulation (WebGL 2.0) can be enabled via URL parameters :  [`http://localho
...
+2. Remove the `signing` configuration block
+```
+// signing {
+//  useGpgCmd()
+// ...
+// }
+```
+You can build and install a libGDX release version to your local Maven repository with the current version declared in the `gradle.properties` by running:
+```
+gradlew.bat -P RELEASE publishToMavenLocal
```
Comment (Berstanio) [inline _pages/dev/-contributing/from_source.md:120]:
```suggestion
gradlew.bat -PRELEASE publishToMavenLocal
```
I guess the whitespace is an error?

**Review by Berstanio — COMMENTED** [review]
Maybe some documentation on how to publish to a custom maven repo would be good to add? So that `override.gradle` is documented?
Through that one could also avoid editing the signing configuration, since you can simply disable it in the `override.gradle` file. This avoids messing with the libGDX files and makes upgrading versions way easier without potential merge conflicts.

File: _pages/dev/-contributing/from_source.md (hunk 1 above)
Comment (obigu) [reply _pages/dev/-contributing/from_source.md:120]: Both work but I-m happy to change it to the more standard no space syntax

**Comment by obigu** [comment]
> Maybe some documentation on how to publish to a custom maven repo would be good to add? So that `override.gradle` is documented? Through that one could also avoid editing the signing configuration, since you can simply disable it in the `override.gradle` file. This avoids messing with the libGDX files and makes upgrading versions way easier without potential merge conflicts.

Great suggestion, didn't know it existed, it has been changed.

**Pull request merged**